---
subtitle: Code Snippets
---

### Passed

#### Passed Example 1

Coming soon

```html
Code snippet
```

### Failed

#### Failed Example 1

Coming soon

```html
Code snippet
```

### Inapplicable

#### Inapplicable Example 1

Coming soon

```html
Code snippet
```
